test/tests/functional/pbs_hook_execjob_prologue.py

```diff
@@ -252,13 +252,14 @@ def test_prologue_hook_does_not_execute_twice_on_pbsdsh(self):
 
         j = Job(PBSROOT_USER, {'Resource_List.select': '2:ncpus=1',
                                'Resource_List.place': 'scatter'})
-        j.create_script('pbsdsh  hostname')
+        j.create_script('pbsdsh  hostname\nsleep 10\n')
```

Comment:
I think you need to add shebang as on some platform (ubuntu) you might face problems running this.